### PR TITLE
Add sky-input-box to Docs and code examples

### PR DESCRIPTION
**Pull request by bradenbiz**
and update outdated vscode settings.json file

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,11 +1,7 @@
 // Place your settings in this file to overwrite default and user settings.
 {
     // Columns at which to show vertical rulers
-    "editor.rulers": [100],
-
-    // Controls after how many characters the editor will wrap to the next line. Setting this to 0 turns on viewport width wrapping (word wrapping). Setting this to -1 forces the editor to never wrap.
-    "editor.wordWrap": "wordWrapColumn",
-    "editor.wordWrapColumn": 100,
```
Comment (bradenbiz) [inline .vscode/settings.json:8]: these rules are remnants of skyux2, so I removed them.

File: package.json [hunk 2]
```diff
@@ -38,35 +38,35 @@
...
+    "@skyux/modals": "4.5.5",
     "@skyux/omnibar-interop": "4.0.1",
-    "@skyux/popovers": "4.5.1",
-    "@skyux/router": "4.1.0",
-    "@skyux/tabs": "4.6.4",
-    "@skyux/theme": "4.15.5",
+    "@skyux/popovers": "4.5.3",
+    "@skyux/router": "4.2.2",
+    "@skyux/tabs": "4.6.7",
+    "@skyux/theme": "4.17.0",
```
Comment (bradenbiz) [inline package.json:69]: all of these are the result of `skyux upgrade`

File: src/app/public/plugin-resources/code-examples/inline-form/basic/inline-form-demo.component.html [hunk 3]
```diff
@@ -37,19 +37,21 @@
...
+          First name
+        </label>
+        <input
+          class="sky-form-control"
+          formControlName="myFirstName"
+          skyId
+          type="text"
+          #firstNameInput="skyId"
+        />
+      </sky-input-box>
```
Comment (bradenbiz) [inline src/app/public/plugin-resources/code-examples/inline-form/basic/inline-form-demo.component.html:54]: for this and all of the following, all i did was surround the `<label>` and `<input>` with a `sky-input-box`

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-inline-form?_sp.spa=pr-addInputBoxToDemos](https://developer.blackbaud.com/skyux-inline-form?_sp.spa=pr-addInputBoxToDemos)

File: src/app/public/plugin-resources/code-examples/inline-form/custom-buttons/inline-form-demo.component.html [hunk 4]
```diff
@@ -37,20 +37,22 @@
...
+          First name
+        </label>
+        <input
+          class="sky-form-control"
+          formControlName="myFirstName"
+          required
+          skyId
+          type="text"
+          #firstNameInput="skyId"
+          />
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/plugin-resources/code-examples/inline-form/custom-buttons/inline-form-demo.component.html]: Extra tab here for the closing bracket.

**Review by Blackbaud-AlexKingman — COMMENTED** [review]
One minor thing.

File: src/app/public/plugin-resources/code-examples/inline-form/custom-buttons/inline-form-demo.component.html (hunk 4 above)
Comment (bradenbiz) [reply src/app/public/plugin-resources/code-examples/inline-form/custom-buttons/inline-form-demo.component.html]: fixed in last commit!

**Review by Blackbaud-AlexKingman — APPROVED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-inline-form?_sp.spa=pr-addInputBoxToDemos](https://developer.blackbaud.com/skyux-inline-form?_sp.spa=pr-addInputBoxToDemos)

**Pull request merged**